HISTORY.rst

```diff
@@ -17,6 +17,7 @@ Bug tracker at https://github.com/giampaolo/psutil/issues
 - #715: don't crash at import time if cpu_times() fail for some reason.
 - #717: [Linux] Process.open_files fails if deleted files still visible.
 - #724: [FreeBSD] virtual_memory().total is slightly incorrect.
+- #863: [Windows] memory_map truncates addresses above 32 bits
```

Comment:
Mmmm looks like you based off this PR from `jhumble:863-fix-64bit-memory_map`. Also looks like you're trying to merge this into `giampaolo:0-wiz-0-master` branch. You should make a PR whch is based off from psutil latest `master` branch instead.